src/server/game/Entities/Player/Player.cpp

```diff
@@ -26085,6 +26086,12 @@ void Player::_LoadSkills(PreparedQueryResult result)
         } while (result->NextRow());
     }
 
+    // Learn skill rewarded spells after all skills have been loaded to prevent learning a skill from them before its loaded with proper value from DB
+    for (auto& skill : loadedSkillValues)
+    {
+        learnSkillRewardedSpells(skill.first, skill.second);
```

Comment:
What exactly is SkillRewardedSpells?

Reply:
Something like this: https://wowgaming.altervista.org/aowow/?skill=124
**Spells**, e. g.  **Cultivation (20552)**,  which are automatically learned when getting a **skill**,  e. g. **Tauren Racial (124)**

Reply:
I really hate the distinction that they make between skills and spells. It's basically a passive spell which is a combination of a class + race. Thanks for clarifying though!